service/lib/agama/dbus/storage/manager.rb

```diff
@@ -22,14 +22,15 @@
 require "dbus"
 require "yast"
 require "y2storage/storage_manager"
+require "agama/dbus/storage/proposal"
+require "agama/dbus/storage/proposal_settings_conversion"
+require "agama/dbus/storage/volume_conversion"
+require "agama/storage/volume_templates_builder"
```

Comment:
why you change ordering? alphabetical one is usually the best one

Reply:
To be honest, we have not followed any specific criteria yet. I take note, because it is something to consider in general.

Reply:
well, I usually use alphabetic one as I can use simple stuff like in vim just select lines and use `:sort` and it is done.

Reply:
Let's sort them.